nodes/modifier_change/offset_line.py

```diff
@@ -160,7 +176,7 @@ def get_middle_points(item_point):
         
         faces_out.extend(new_edges)
 
-    verts_out = [i.to_3d()[:] for i in verts_out]
+    verts_out = [(i.to_3d() + Vector((0,0,z_co[c])))[:] for c,i in enumerate(verts_out)]
```

Comment:
```diff
- verts_out = [(i.to_3d() + Vector((0,0,z_co[c])))[:] for c,i in enumerate(verts_out)]
+ verts_out = [(v.x, v.y, z_co[idx]) for idx, v in enumerate(verts_out)]
```

Reply:
the convention is to use i or idx for the index of the enumerator. else this promotes confusion to the reader (even you... in 2 months time). and we tend to use `v` to indicate that an element is a vector (or similar triplet)

i suggest using `v.x, v,y, ...` here, but you can use `v[0], v[1], ....`  . bracket lookup might be faster, but v.x and v.y are a little more meaningful.

We like to avoid the use of Vector entirely, because casting from [x,y,z] to Vector([x,y,z]) introduces a slowdown -- I don't think this is a problem now, but to increase speed if it's ever a serious problem it's try implementing this using the verts/edges converted to a bmesh representation.

Reply:
Ok, I got it.